pkg/restic/restic.go

```diff
@@ -342,26 +342,38 @@ func SnapshotIDFromBackupLog(output string) string {
 }
 
 // SnapshotStatsFromBackupLog gets the Snapshot file count and size from Backup Command log
-func SnapshotStatsFromBackupLog(output string) (fileCount string, backupSize string) {
+func SnapshotStatsFromBackupLog(output string) (fileCount string, backupSize string, phySize string) {
 	if output == "" {
-		return "", ""
+		return "", "", ""
 	}
 	logs := regexp.MustCompile("[\n]").Split(output, -1)
 	// Log should contain "processed %d files, %.3f [Xi]B in mm:ss"
```

Comment:
nit: please also add comment for logs containing `Added to the repo....`

Reply:
done